src/js/components/Institutions.jsx

```diff
@@ -70,25 +70,12 @@ export const renderStatus = (
 export const renderViewButton = (code, institutionId, period) => {
   let buttonText
 
-  switch (code) {
-    // not-started
-    case 1:
-      buttonText = 'Begin filing'
-      break
-    // in progress
-    case 2:
-      buttonText = 'View current filing'
-      break
-    // completed
-    case 3:
-      buttonText = 'View completed filing'
-      break
-    // cancelled
-    case 4:
-      buttonText = 'Begin filing'
-      break
-    default:
-      buttonText = 'Begin filing'
+  if(code === STATUS.CREATED){
+    buttonText = 'Begin filing'
+  }else if(code === STATUS.SIGNED){
+    buttonText = 'View completed filing'
+  }else{
```

Comment:
Can we remove the last `else` by setting `let buttonText = 'View current filing'` above?